run-build.sh

```diff
@@ -30,18 +30,17 @@ else
 fi
 
 # Node version
-source /.nvm/nvm.sh
-export NODE_VERSION=0.12.2
-if [ -f .nvmrc ]
-then
-	if nvm use; then
-		echo "Set node from .nvmrc"
-		export NODE_VERSION=$(cat .nvmrc)
-	else
-		echo "Error setting node version from .nvmrc file. Unsupported version?"
-		echo "Will use default version ($NODE_VERSION)"
-	fi
+source $HOME/.nvm/nvm.sh
+export NODE_VERSION=4
+if [ -f .nvmrc ]; then
+  nvm install $(cat .nvmrc)
```

Comment:
I'm not sure if that's very efficient. Does `nvm install` do nothing if the versions are the same?